docs/getting-started/README.md

```diff
@@ -38,13 +38,14 @@ make GOFLAGS="-buildvcs=false" CC=clang
 
 **Runtime Dependencies**
 
-Download [geoip.dat](https://github.com/v2ray/geoip/releases/latest) and [geosite.dat](https://github.com/v2fly/domain-list-community/releases/latest) to `/usr/local/share/dae/`.
+For traffic splitting, Dae relies on the following data sources, [geoip.dat](https://github.com/v2ray/geoip/releases/latest) and [geosite.dat](https://github.com/v2fly/domain-list-community/releases/latest).
 
-```
+```shell
 mkdir -p /usr/local/share/dae/
 pushd /usr/local/share/dae/
 curl -L -o geoip.dat https://github.com/v2ray/geoip/releases/latest/download/geoip.dat
 curl -L -o geosite.dat https://github.com/v2ray/domain-list-community/releases/latest/download/dlc.dat
+curl -L -o config.dae https://github.com/v2rayA/dae/raw/main/example.dae
```

Comment:
This file should be downloaded to `/etc/dae/`. Or you can move it after `popd`.

Reply:
Resolved in [eb8dc6c](https://github.com/v2rayA/dae/pull/25/commits/eb8dc6c4554d89391f4f7d20b9b4a0000a98126e)